RideService.Entities/Ride.cs

```diff
@@ -98,7 +98,7 @@ public int DaysSinceBreakdown()
             {
                 return 0;
             }
-            TimeSpan timeSpan = DateTime.Today - latestBreakdown;
+            TimeSpan timeSpan = DateTime.Now - latestBreakdown;
```

Comment:
Hvorfor har du lavet den om til at bruge Now i stedet for Today??

Reply:
Jeg tænkte at det ville fungere bedre på den måde.